jest/src/main/java/io/searchbox/client/http/JestHttpClient.java

```diff
@@ -114,6 +137,10 @@
         }
     }
 
+    private boolean isHttpSuccessful(int httpCode) {
+        return (httpCode / 100) == 2;
```

Comment:
Avoid logic based on division. Race conditions around rounding errors in floating point is too weird. Also, not everyone understands the logic of HTTP response codes. A more readable implementations would be:

```
        return httpCode >= StatusCode.SC_OK && httpCode < StatusCode.SC_MULTIPLE_CHOICES;
```